test/amp.test.ts

```diff
@@ -0,0 +1,77 @@
+import * as cdk from 'aws-cdk-lib';
+import * as blueprints from '../lib';
+
+describe('Unit tests for AMP addon', () => {
+    
+    test("Stack creation succeeds", () => {
+        const app = new cdk.App();
+
+        const blueprint = blueprints.EksBlueprint.builder();
+
+        blueprint.account("123567891").region('us-west-1')
+        .addOns(new blueprints.addons.AwsLoadBalancerControllerAddOn())
+        .addOns(new blueprints.addons.CertManagerAddOn())
+        .addOns(new blueprints.addons.AdotCollectorAddOn())
+        .addOns(
+            new blueprints.addons.AmpAddOn({
+                ampPrometheusEndpoint: "test",
+                ampRules: {
+                    ampWorkspaceArn: "test",
+                    ruleFilePaths: [
+                        __dirname + "/resources/recording-rules-test.yml",
+                    ]
+                }
+            })
+        )
+        .build(app, 'amp-addon-stack-succeeds');
+
+        expect(blueprint).toBeDefined();
+    });
+
+    test("Stack creation fails due to missing AdotCollectorAddOn", () => {
```

Comment:
I think this test is not required because we have a dependency with ADOT Addon in AmpAddon. But if you prefer to keep it im good.

Reply:
done